sdk/storage/README.md

```diff
@@ -167,11 +167,11 @@ async function main() {
   // SharedKeyCredential is only avaiable in Node.js runtime, not in browsers
   const sharedKeyCredential = new SharedKeyCredential(account, accountKey);
 
-  // Use RawTokenCredential with OAuth token.  You can find more
+  // Use SimpleTokenCredential with OAuth token.  You can find more
   // TokenCredential implementations in the @azure/identity library
```

Comment:
About description "from @azure/identity". 

`SimpleTokenCredential` is from `@azure/core-http` while other TokenCredential implmenetations from `@azure/identity`.  We'd better make the description clearer.

Reply:
This should be taking [TokenCredential](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-auth/src/tokenCredential.ts) not SimpleTokenCredential

Reply:
I opened issue https://github.com/Azure/azure-sdk-for-js/issues/4968 to remove SimpleTokenCredential from core-auth

Reply:
Okay. `SimpleTokenCredential` should be removed from core-auth and won't be exposed in our libraries.

@XiaoningLiu, you've recently added some tests that leverage `SimpleTokenCredential` a few days ago.
Should we implement this `SimpleTokenCredential` class on top of [TokenCredential](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-auth/src/tokenCredential.ts) in the [test/utils/index.ts](https://github.com/Azure/azure-sdk-for-js/blob/feature/storage/sdk/storage/storage-blob/test/utils/index.ts#L68) to leverage it just for the tests and not expose it anywhere else from the library?

Reply:
@HarshaNalluru Sounds good. One question about removing `SimpleTokenCredential`. This is the raw or last way for customers to create a TokenCredential. Does @azure/identity work with Node.js & browsers to get a TokenCredential?

Reply:
[@azure/identity](https://www.npmjs.com/package/@azure/identity) provides many types of TokenCredentials for both Node.js & browsers.
Created and issue to test and add samples - https://github.com/Azure/azure-sdk-for-js/issues/5051